src/Helpers.php

```diff
@@ -503,7 +503,7 @@ public static function dompdf_debug($type, $msg)
     public static function record_warnings($errno, $errstr, $errfile, $errline)
     {
         // Not a warning or notice
-        if (!($errno & (E_WARNING | E_NOTICE | E_USER_NOTICE | E_USER_WARNING))) {
+        if (!($errno & (E_WARNING | E_NOTICE | E_USER_NOTICE | E_USER_WARNING | E_STRICT | E_DEPRECATED | E_USER_DEPRECATED))) {
```

Comment:
I added these levels here in case if any piece of the code between our `try..finally` block triggers a deprecation in the future.